src/Microsoft.AspNetCore.HttpsPolicy/HstsMiddleware.cs

```diff
@@ -19,10 +20,16 @@ public class HstsMiddleware
     {
         private const string IncludeSubDomains = "; includeSubDomains";
         private const string Preload = "; preload";
+        private static readonly string[] LocalhostStrings =
+        {
+            "localhost",
+            "127.0.0.1", // ipv4
+            "[::1]" // ipv6
+        };
```

Comment:
@Tratcher anything else for this list?

Reply:
We gave the option for people to specify other domains to block. I think these three are the major ones.